velox/exec/tests/TableScanTest.cpp

```diff
@@ -2291,6 +2291,76 @@ TEST_F(TableScanTest, path) {
       op, {filePath}, fmt::format("SELECT '{}', * FROM tmp", pathValue));
 }
 
+TEST_F(TableScanTest, fileSizeAndModifiedTime) {
```

Comment:
TableScanTest

Reply:
hey @aditi-pandit would you like update your PR following our discussion or still leverage my PR ?

Reply:
@gaoyangxiaozhu : Apologize. Didn't get a chance to get to this yesterday. Have updated it now.